custom_components/plugwise/config_flow.py

```diff
@@ -339,23 +339,23 @@ async def async_step_init(
 
         data = {
             vol.Optional(
-                timedelta(seconds=CONF_SCAN_INTERVAL),
+                timedelta(seconds=float(CONF_SCAN_INTERVAL)),
                 default=interval,
             ): timedelta,
         }  # pw-beta
 
         if coordinator.api.smile_type != "thermostat":
             return self.async_show_form(step_id="init", data_schema=vol.Schema(data))
 
-        data = {
-            vol.Optional(
-                timedelta(seconds=CONF_SCAN_INTERVAL),
-                default=interval,
-            ): timedelta,
-            vol.Optional(
-                CONF_HOMEKIT_EMULATION,
-                default=self.config_entry.options.get(CONF_HOMEKIT_EMULATION, False),
-            ): cv.boolean,
-        }  # pw-beta
+        data.update(
```

Comment:
@bouwew somehow (probably because we reuse the var) mypy complains, if we just 'extend' the dict it doesn't (could have gone the `data[len(data)]` way, but this is more convenient

Reply:
Yes, that's fine.